WORKSPACE

```diff
@@ -188,8 +188,6 @@ maven_install(
         # https://github.com/bazelbuild/rules_jvm_external/issues/59
         "junit:junit:4.12",
         "org.jetbrains.kotlin:kotlin-test:1.3.21",
-        # https://github.com/bazelbuild/rules_jvm_external/issues/101
-        "com.digitalasset:damlc:jar:osx:100.12.1",
```

Comment:
Is it worth it to find another artifact with a classifier to keep this test?

Reply:
Good point, replacing with another asset with a classifier.